src/cli/cli.ts

```diff
@@ -284,6 +285,7 @@ async function openPage(context: BrowserContext, url: string | undefined): Promi
 
 async function open(options: Options, url: string | undefined) {
   const { context } = await launchContext(options, false);
+  (context as any)._extendInjectedScript(consoleApiSource.source);
```

Comment:
These types should just work right? Because this is the internal context not the types we generate.

Reply:
Good point!